File: app/src/main/java/com/drivingschoolrwandaapp/models/entities/Timezone.java
package com.drivingschoolrwandaapp.models.entities;

import com.google.gson.annotations.SerializedName;

import java.io.Serializable;

public class Timezone implements Serializable {
    @SerializedName("id")
    private int id;

    @SerializedName("timezoneName")
    private String timezoneName;

    @SerializedName("utcOffset")
    private String utcOffset;

    @SerializedName("countryName")
    private String countryName;

    // Getters and Setters
    public int getId() { return id; }
    public String getTimezoneName() { return timezoneName; }
    public String getUtcOffset() { return utcOffset; }
    public String getCountryName() { return countryName; }
}

File: app/src/main/java/com/drivingschoolrwandaapp/models/entities/Timezone.kt
package com.drivingschoolrwandaapp.models.entities

import com.google.gson.annotations.SerializedName
import java.io.Serializable

data class Timezone(
    @SerializedName("id") var id: Int = 0,
    @SerializedName("timezoneName") var timezoneName: String? = null,
    @SerializedName("utcOffset") var utcOffset: String? = null,
    @SerializedName("countryName") var countryName: String? = null
) : Serializable
